sql/installation_metadata.sql

```diff
@@ -0,0 +1,9 @@
+CREATE OR REPLACE FUNCTION _timescaledb_internal.generate_uuid() RETURNS UUID
+AS '@MODULE_PATHNAME@', 'ts_uuid_generate' LANGUAGE C VOLATILE STRICT;
+
+-- Insert uuid and install_timestamp on database creation.
+-- Don't create exported_uuid because this shouldn't get called on a create
```

Comment:
Hm, I thought I changed the second sentence of this comment. It's not particularly descriptive right now. It should read something like "Don't create exported_uuid because it gets exported and installed during pg_dump, which would cause a conflict."